js/common/categories.js

```diff
@@ -0,0 +1,25 @@
+window.onload = () => {
```

Comment:
It would be appreciated if the categories.js file be treated as the main "controller" for the categories page. Specifically, all behavior for the categories page should be defined, used, and/or controlled from this file.

This is true for all .js files that have the same name as the page that they are dealing with. They must act as controllers for that view/page.